crates/aptos/src/common/types.rs

```diff
@@ -1669,3 +1671,61 @@ pub struct RotationProofChallenge {
     pub current_auth_key: AccountAddress,
     pub new_public_key: Vec<u8>,
 }
+
+/// Common options for constructing an entry function transaction payload.
+#[derive(Debug, Parser)]
+pub struct EntryFunctionArguments {
+    /// Function name as `<ADDRESS>::<MODULE_ID>::<FUNCTION_NAME>`
+    ///
+    /// Example: `0x842ed41fad9640a2ad08fdd7d3e4f7f505319aac7d67e1c0dd6a7cce8732c7e3::message::set_message`
+    #[clap(long)]
+    pub function_id: MemberId,
+
+    /// Arguments combined with their type separated by spaces.
+    ///
+    /// Supported types [u8, u16, u32, u64, u128, u256, bool, hex, string, address, raw, vector<inner_type>]
```

Comment:
Does this need Object etc.. support

Reply:
Yeah. It's likely not as high priority so later :)